contract/deploy.js

```diff
@@ -60,7 +64,7 @@ const installBundle = async (pathResolve, zoe, board) => {
  * Object, wallet: ERef<Object>, faucet: ERef<Object>}>} homePromise
  * @param {DeployPowers} powers
  */
-const deployContract = async (homePromise, { pathResolve }) => {
+const deployContract = async (homePromise, { pathResolve, publishBundle }) => {
```

Comment:
Quick question, will older SDKs simply not provide `publishBundle` in the `deployPowers`? In which case the loadgen could feature test for presence and fallback to installing the bundle directly if it's missing?

Reply:
Yes, `publishBundle` is absent in prior versions of `agoric-cli`.